pwiz_tools/Skyline/Controls/Graphs/DetectionsPlotPane.cs

```diff
@@ -396,10 +404,31 @@ protected virtual void AddLabels()
                         Title.Text = Resources.DetectionPlotPane_EmptyPlotError_Label;
                         break;
                 }
+                var scaleFactor = CalcScaleFactor();
+                SizeF titleSize;
+                using (var g = GraphSummary.CreateGraphics())
+                {
+                    titleSize = Title.FontSpec.BoundingBox(g, Title.Text, scaleFactor);
+                }
+                var subtitleLocation = new PointF(
+                    (Rect.Left + Rect.Right) / (2 * Rect.Width),
+                    (Rect.Top + Margin.Top * (1 + scaleFactor) + 2*titleSize.Height) / Rect.Height);
+
+                var subtitle = new TextObj(message, subtitleLocation.X, subtitleLocation.Y,
+                    CoordType.PaneFraction, AlignH.Center, AlignV.Center)
+                {
+                    IsClippedToChartRect = true,
+                    ZOrder = ZOrder.E_BehindCurves,
+                    FontSpec = GraphSummary.CreateFontSpec(Color.Black),
+                };
+                subtitle.FontSpec.Size = Title.FontSpec.Size * 0.75f;
+                GraphObjList.Add(subtitle);
+            }
             else
             {
                 Title.Text = string.Empty;
-                YAxis.Title.Text += @" (" + String.Format(CultureInfo.CurrentCulture, Settings.YScaleFactor.Label, Settings.TargetType.Label) + @")";
+                YAxis.Title.Text = Resources.DetectionPlotPane_YAxis_Name +
+                                   @" (" + String.Format(CultureInfo.CurrentCulture, Settings.YScaleFactor.Label, Settings.TargetType.Label) + @")";

```

Comment:
Parentheses are not the same in all languages. Why not just include these in YAxis_Name, and use string.Format() with:

Name ({0})

So that a translator can change the parentheses as well as the ordering, if that is every appropriate?